Modules/MFT/src/QcMFTClusterTask.cxx

```diff
@@ -308,7 +307,8 @@ void QcMFTClusterTask::monitorData(o2::framework::ProcessingContext& ctx)
     if (oneCluster.getPatternID() != o2::itsmft::CompCluster::InvalidPatternID && !mDict->isGroup(oneCluster.getPatternID())) {
       mClusterSizeSummary->Fill(mDict->getNpixels(oneCluster.getPatternID()));
     } else {
-      o2::itsmft::ClusterPattern patt(patternIt);
+      auto patternIt2 = clustersPattern.begin();
```

Comment:
One should start from `clustersPattern.begin();` once and then increment the same iterator in the cluster loop. Could you move this line before https://github.com/AliceO2Group/QualityControl/pull/1434/files#diff-5a078c880ac63735b04899b111617d16021b9589832e1f8a90cd601273318c73R296? Or, you can simply reuse the patternIt after resetting it to `clustersPattern.begin()` instead of adding a new iterator.

Reply:
Hi Ruben, thanks a lot, I edited the code.